### PR TITLE
Enable parallel feature activiation for JCA / Connectors

### LINKED ISSUE #14356: Enable parallel feature activiation for JCA / Connectors
Go through all private and public JCA / Connectors features and enable parallel feature activation using `WLP-Activation-Type: parallel`

### PR DESCRIPTION
**Pull request by KyleAure**
Fixes #14356

### CODE REVIEW

**Comment by KyleAure** [comment]
Requesting a #build to see if there are any concerns with performing parallel activation.

File: dev/com.ibm.websphere.appserver.features/visibility/auto/com.ibm.websphere.appserver.jca.resourcedefinition.jms-2.0.feature [hunk 1]
```diff
@@ -10,3 +10,4 @@ IBM-Provision-Capability: osgi.identity; filter:="(&(type=osgi.subsystem.feature
 IBM-Install-Policy: when-satisfied
 kind=ga
 edition=base
+WLP-Activation-Type: parallel
```
Comment (jhanders34) [inline dev/com.ibm.websphere.appserver.features/visibility/auto/com.ibm.websphere.appserver.jca.resourcedefinition.jms-2.0.feature:13]: Do you want to do io.openliberty.connectors.resourcedefinition.messaging-3.0.feature as well?

File: dev/com.ibm.websphere.appserver.features/visibility/protected/io.openliberty.appserver.connectors-2.0.feature [hunk 2]
```diff
@@ -13,3 +13,4 @@ IBM-API-Package: jakarta.resource; type="spec", \
 -bundles=io.openliberty.jakarta.connectors.2.0; location:="dev/api/spec/,lib/"; mavenCoordinates="jakarta.resource:jakarta.resource-api:2.0.0"
 kind=beta
 edition=base
+WLP-Activation-Type: parallel
```
Comment (jhanders34) [inline dev/com.ibm.websphere.appserver.features/visibility/protected/io.openliberty.appserver.connectors-2.0.feature:16]: Do you want to also update com.ibm.websphere.appserver.javax.connector-1.6.feature and 
com.ibm.websphere.appserver.javax.connector-1.7.feature?

File: dev/com.ibm.websphere.appserver.features/visibility/private/com.ibm.websphere.appserver.internal.jca-1.6.feature [hunk 3]
```diff
@@ -22,3 +22,4 @@ IBM-API-Package: \
   com.ibm.ws.jca.feature
 kind=ga
 edition=base
+WLP-Activation-Type: parallel
```
Comment (jhanders34) [inline dev/com.ibm.websphere.appserver.features/visibility/private/com.ibm.websphere.appserver.internal.jca-1.6.feature:25]: Do you also want to update com.ibm.websphere.appserver.javax.connector.internal-1.6.feature?

**Review by jhanders34 — CHANGES_REQUESTED** [review]

File: dev/com.ibm.websphere.appserver.features/visibility/auto/com.ibm.websphere.appserver.jca.resourcedefinition.jms-2.0.feature (hunk 1 above)
Comment (KyleAure) [reply dev/com.ibm.websphere.appserver.features/visibility/auto/com.ibm.websphere.appserver.jca.resourcedefinition.jms-2.0.feature:13]: @dazavala is working on the messaging-3.0 feature.  I'll leave the messaging features out of this update for now.

Comment (KyleAure) [reply dev/com.ibm.websphere.appserver.features/visibility/auto/com.ibm.websphere.appserver.jca.resourcedefinition.jms-2.0.feature:13]: Nevermind, I thought was was referring to the public feature.  Updated :)

**Review by jhanders34 — APPROVED** [review]

**Comment by KyleAure** [comment]
#build 
#libbybot

**Pull request merged**